packages/languages/src/browser/language-client-contribution.ts

```diff
@@ -228,4 +232,28 @@ export abstract class BaseLanguageClientContribution implements LanguageClientCo
         });
     }
 
+    /**
+     * Return the id of the "restart" command for this language client.
+     */
+    private restartCommandId(): string {
+        return `languages.${this.id}.restart`;
+    }
+
+    /**
+     * Register a command that lets the user restart the language server this
+     * client is connected to.
+     */
+    protected registerRestartCommand(): void {
+        this.registry.registerCommand({
+            id: this.restartCommandId(),
+            label: `Restart ${this.name} language server`,
```

Comment:
Maybe `{this.name}: Restart Server` to align with other commands?

Reply:
Ok, I changed it to `${this.name}: Restart Language Server`.